eslint.config.js

```diff
@@ -0,0 +1,28 @@
+import js from '@eslint/js'
+import globals from 'globals'
```

Comment:
**issue**: Consider using consistent naming conventions for imports, such as `@typescript-eslint` instead of `typescript-eslint`, to avoid confusion and align with common practices.